api/package/npm/ownership.js

```diff
@@ -0,0 +1,9 @@
+const { NpmOwnershipRequest } = require('../../../types/Package')
+const { Base200Response } = require('../../../common/reply')
+
+module.exports = {
+  body: NpmOwnershipRequest,
+  response: {
+    200: Base200Response
```

Comment:
is the idea that no data will be returned in this req, but rather based on status code the UI would determine if we could proceed to dashboard and then at dashboard we'd fetch owned packages?

Reply:
yes -- idea is that this request will not return all the packages, but just success: t/f. if success true, the client can fetch owned packages back on the maintainer dash.